src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -17381,6 +17460,79 @@ public static void Main()
         }
 
         [Fact]
+        public void OverriddenPropertyWithDifferentTupleNamesInReturn()
```

Comment:
InheritFromMetadataWithDifferentNames seems to be doing the right thing.

---
In reply to: [81822771](https://github.com/dotnet/roslyn/pull/14229#discussion_r81822771) [](ancestors = 81822771)